miso-web/src/main/webapp/scripts/hot_sample.js

```diff
@@ -498,7 +518,7 @@ HotTarget.sample = (function() {
             pack : function(sam, flat, errorHandler) {
               sam.dnaseTreated = flat.dnaseTreated === 'True';
             },
-            include : show['Stock'] && config.targetSampleClass.dnaseTreatable
+            include : show['Stock'] && (config.targetSampleClass.dnaseTreatable || (config.create && show['Aliquot']))
```

Comment:
Not all stocks are DNAse treatable though. I think DNAse treatable needs to be a config parameter and the controller should set it to true if the class is a stock and DNAse treatable or an aliquot and there exists a non-archived parent stock that's treatable.